### ISSUE TITLE
mindmap was loss

### ISSUE THREAD

**Issue by linxylz** [marc0l92/joplin-plugin-plantUML#7]
Some times ,when I switched joplin's built-in editor  , the mindMap definition was lost, just left  “Copy image” and “Copy image address”。
Such as:

```plantuml
@startmindmap
* Debian
** Ubuntu
*** Linux Mint
*** Kubuntu
*** Lubuntu
*** KDE Neon
** LMDE
** SolydXK
** SteamOS
** Raspbian with a very long name
*** <s>Raspmbc</s> => OSMC
*** <s>Raspyfi</s> => Volumio
@endmindmap
```

Just left

- Copy image
- Copy image address

![image](https://user-images.githubusercontent.com/93452239/149646351-ff44974c-21fd-4403-96c5-e825527600a2.png)

_____________________________________
Environment
Joplin version: 2.6.10
Platform: Windows
OS specifics: Windows 10  LTSC, Build 17763.2452

**Comment by kdopen** [comment]
I just experienced this with a (large) ERD :(

**Comment by marc0l92** [comment]
This is a limitation of Joplin that I'm not able to fix. Everything that is not supported by the RichText editor is deleted if you try to modify the note in that mode.
You could experience the same issue with all the other plugins that are generating custom markdown syntax.

What I can do is to create a backup of the diagrams so you can restore them if needed.

I'm just not sure about the strategy to keep those backup:
- delete the backup when you close the application
- delete the backup 2 weeks after the creation
- delete the oldest backup when you reach 10MB of backups

What do you suggest?

**Comment by marc0l92** [comment]
I find a way to fix this issue without the need of a workaround.
You can find the fix in version 1.9.0 of this plugin.
